### DISCUSSION TITLE
Task 12 perturbation

### DISCUSSION THREAD

**Discussion by AprilSweettooth** [Part-II-Computational-Physics/.github#37] [Q&A] [answered]
Does the small perturbation in Task12 mean adding a sigmaX operator with coefficient in magnitude of h?

**Comment by JNHallen** [comment, marked as answer]
There are different perturbations one could imagine, but for it to count as a small perturbation it should have magnitude $\ll h$. Adding a sigmaX operator will not select the all-up state (in the Z-basis) as the ground state at small $h$, so that would not be the right choice. You need to include a term in the Hamiltonian that will give the all-up state a lower energy than the all down state.

**Reply by AprilSweettooth** [reply]
Thanks, I see. I though as long as I use Hamiltonian like $m^z = \frac{1}{N} \sum_i \sigma_i^z - h \sum_i \sigma_i^x$ then as h goes to 0 the second term would vanish and we get an all z direction pointing state. But I suppose we need an operator that differentiates up and down state?

**Reply by JNHallen** [reply]
Exactly! In the transverse field Ising model without the transverse field all states come in degenerate pairs, as flipping all spins do not change the energy. What you want is to include some term that breaks this degeneracy in your Hamiltonian. One way to do that would be to add a longitudinal field such as 
$\delta {\cal H} = -g \sum_i \sigma_i^z$
with $g \ll h$.
